### PR TITLE
Update Prisma example

### PR DESCRIPTION
**Pull request by cobrexus**

### CODE REVIEW

File: examples/with-prisma/.gitignore [hunk 1]
```diff
@@ -1,34 +0,0 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (ijjk) [inline examples/with-prisma/.gitignore:1]: Why was this file removed?

**Review by ijjk — APPROVED** [review]

**Review by leerob — APPROVED** [review]
